src/js/components/PageHeader.js

```diff
@@ -108,20 +113,27 @@ class PageHeader extends React.Component {
               </div>
             </div>
           </div>
-          {children}
+          {navigationTabs}
         </div>
       </div>
     );
   }
 }
 
 PageHeader.defaultProps = {
-  actionButtons: []
+  actionButtons: [],
+  className: 'container container-fluid container-pod container-pod-short flush flush-top',
+  dividerClassName: 'container-pod container-pod-short flush-top flush-bottom container-pod-divider-bottom container-pod-divider-bottom-align-right container-pod-divider-inverse',
+  iconClassName: 'icon icon-large icon-image-container icon-app-container',
+  mediaWrapperClassName: 'media-object-spacing-wrapper media-object-spacing-narrow media-object-offset',
+  subTitleClassName: 'emphasize',
+  titleClassName: 'flush inverse'
```

Comment:
defaultProps are back again! Let's get rid of them again

(c.f. https://github.com/dcos/dcos-ui/pull/118/files/d9a9be346cf5f776ece2e3180785d82d06deaed6#r63564663)